test/xema/struct_test.exs

```diff
@@ -13,6 +13,14 @@ defmodule Xema.StructTest do
     defstruct [:foo]
   end
 
+  defmodule Baz do
+    use Xema
+
+    xema do
+      field :foo_bar, [:list], default: []
```

Comment:
Types has just to be set in a list if we want multiple possible types (e.g. `[:list, nil]`).
```suggestion
      field :foo_bar, :list, default: []
```